scripts/generate-api-docs.js

```diff
@@ -1,7 +1,11 @@
 import { execSync } from "child_process";
 
+const packages = ["react-router", "react-router-dom", "react-router-native"];
+
 execSync(
-  `typedoc --ignoreCompilerErrors --includeDeclarations --excludeExternals --out docs/api build`,
+  `typedoc ${packages
+    .map(pkg => `packages/${pkg}/index.tsx`)
+    .join(" ")} --out docs/api --name "React Router"`,
```

Comment:
So all of the CLI flags we used before were removed, but I actually think this makes a bit more sense. We can generate types directly from the source which is probably fine, but I'm happy to revisit this if we need to.